extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -263,7 +264,6 @@
 }
 
 .i-amphtml-story-bookend-component-set {
-  font-family: 'Roboto', sans-serif !important;
```

Comment:
Could you also remove it here?
https://github.com/Enriqe/amphtml/blob/fb12524726196e461f690ae39b7c31ba6da60230/extensions/amp-story/1.0/amp-story-bookend.css#L304

Thanks!

Reply:
Done.